ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -4,6 +4,12 @@
     <span>
       <form role="search" class="form-inline">
 
+        <span class="btn btn-sm btn-view-nav nav-menu-btn" 
+              ng-show="serverChanged"
+              id="revisionChangedLabel"
+              title="New version of Treeherder has been deployed. Reload to pick up changes.">
+          <span class="fa fa-exclamation-circle">&nbsp;Server Revision Changed</span>
+        </span>
```

Comment:
A few things about this:

1. It's styled as a button, but nothing happens when you click on it. I think we should either remove the styling, or maybe make clicking it actually reload the page (http://stackoverflow.com/questions/3715047/how-to-reload-a-page-using-javascript) perhaps hidden behind a confirmation / alert dialog.
2. Server revision changed is more an internal state. I would say perhaps "Treeherder update available".
3. If you're inserting a font-awesome element via a span, you should close the span before inserting text, otherwise the text looks all funny. Try:

  ```html
  <span class="fa fa-exclamation-circle"/>&nbsp;Treeherder update available
  ```

4. You may also consider using the upload icon here instead of the exclamation: http://fontawesome.io/icon/upload/
5.You should hide this behind an ng-cloak before the page is loaded: https://docs.angularjs.org/api/ng/directive/ngCloak